pandas/io/parsers.py

```diff
@@ -1081,14 +1084,20 @@ def _clean_options(self, options, engine):
         na_values, na_fvalues = _clean_na_values(na_values, keep_default_na)
 
         # handle skiprows; this is internally handled by the
-        # c-engine, so only need for python parsers
+        # c-engine, so only need for python parser
```

Comment:
give more context here. skipsrows can be an integer or a list of ints, make this a bit more clear via comments / code